roles/acme/tasks/download_cert.yml

```
@@ -0,0 +1,13 @@
+---
+- name: Fetch current certificate from https server
+  ansible.community.crypto.get_certificate:
```

As suggested in internal Chat :)
```suggestion
  community.crypto.get_certificate:
```

---

has already been fixed while testing.